core/tests/protractor_desktop/additionalEditorFeatures.js

```diff
@@ -383,6 +385,390 @@ describe('Full exploration editor', function() {
     await users.logout();
   });
 
+  it(
+    'should show merge changes when the changes are not conflicting',
+    async function() {
+      await users.createUser(
+        'user9@editor.com',
+        'user9Editor');
```

Comment:
Here and below, can we try to write statements in fewer lines if it is possible to do so? e.g. you can do `expect(await audioLanguageSelector.getText()).not.toEqual(language);` here without hitting the max line limit.

Reply:
Done.